torch/_dynamo/pgo.py

```diff
@@ -559,6 +564,7 @@ def hit(ty: str) -> DefaultDict[CodeId, CodeState]:
             "string",
             lambda: render_code_state(_CODE_STATE),
         )
+        set_feature_use("pgo", True)
```

Comment:
Maybe pgo.get_code_state.hit? IDK man lol

Reply:
Yeah I feel like simpler is better here for binary signals.